README.md

```diff
@@ -10,83 +9,95 @@ A curated list of Awesome Alfred Workflows.
 [![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
 
 ## Contents
+
 - [Communication](#communication)
 - [Developer](#developer)
-- [Helpers](#helpers)
+- [Text Manipulation](#text-manipulation)
```

Comment:
It was alphabetical before, reason for change?

Reply:
Whoops! Not sure what happened there, my bad. Should have been organized alphabetically, until Other Lists. My next commit should correct this but let me know if you'd prefer it differently